azure-votes-redis/README.md

```diff
@@ -0,0 +1,80 @@
+# Azure Service Operator sample for Azure Cache for Redis
+
+This sample is a demonstration of how to use the Azure Service Operator (ASO) to provision an Azure Cache for Redis,
+and then deploy a web application that uses that managed Redis instance to store its data.
+
+## Prerequisites
+
+To deploy this demo application you'll need the following:
+
+1. A Kubernetes cluster (at least version 1.21) [created and
+   running](https://kubernetes.io/docs/tutorials/kubernetes-basics/create-cluster/),
+   and [`kubectl`](https://kubernetes.io/docs/tasks/tools/#kubectl) configured to talk to it. (You can check your cluster
+   version with `kubectl version`.) This could be a local [Kind cluster](https://kind.sigs.k8s.io/docs/user/quick-start/)
+   or an [Azure Kubernetes Service
+   cluster](https://docs.microsoft.com/en-us/azure/aks/tutorial-kubernetes-deploy-cluster)
+   running in your subscription.
+
+2. An Azure subscription to create Azure resources under.
+
+## Set up Azure Service Operator
+
+ASO lets you manage Azure resources using Kubernetes tools.
+The operator is installed in your cluster and propagates changes to resources there to the Azure Resource Manager.
+[Read more about how ASO works](https://github.com/azure/azure-service-operator#what-is-it)
+
+Follow [these
+instructions](https://github.com/Azure/azure-service-operator/tree/master/v2#installation) to install the ASO v2 operator in your cluster.
+Part of this installs
+the [custom resource definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) for some of the Azure Resources.
+
+### Note: 
+Only a few Custom Resource Definitions will be installed in your cluster when you follow the Azure Service Operator set up instructions. The CRDs required specifically to run this sample will be installed along with the sample.
```

Comment:
> The CRDs required specifically to run this sample will be installed along with the sample.

There's nothing in the YAML file that will result in ASO putting new CRDs into the cluster - the pattern `cache.azure.com/*` needs to be included in the CRD Patterns configuration of ASO when it's installed.

How about this rephrasing:

```suggestion
As you follow the installation instructions for Azure Service Operator, add `cache.azure.com/*` to the configuration of CRD Patterns. (ASO doesn't automatically install all available Custom Resource Definitions, as most users only want a small subset.)
```